src/sentry/tasks/code_owners.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from sentry import features
+from sentry.tasks.base import instrumented_task
+
+logger = logging.getLogger("sentry.tasks.code_owners")
```

Comment:
nit: it seems like we use `codeowners` as one word in many places without the `_`, do we want to be consistent with for this file as well?

Reply:
@MeredithAnya We're calling the feature `Code Owners`. So I'm migrating to using `codeowners` to denote the CODEOWNERS text and `code_owners` for our feature.